server/test/uk/gov/ons/addressIndex/server/controllers/AddressControllerSpec.scala

```diff
@@ -111,6 +118,12 @@ class AddressControllerSpec  extends PlaySpec with Results with AddressIndexCann
       Future.successful(HybridAddresses(Seq.empty, 1.0f, 0))
 
     override def client(): ElasticClient = ElasticClient.local(Settings.builder().build())
+
+    override def logger: Logger = ???
```

Comment:
Those methods shouldn't be public as the user of the interface shouldn't use them.